File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.py
class Solution:
    def sumZero(self, n: int) -> List[int]:
        if(n%2==0):
            count=-1*n
            ans=[x for x in range(count+1,n,2)]
        else:
            start=(n//2)*-1
            end=(n//2)
            ans=[x for x in range(start,end+1)]            
        return ans

File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.cpp
class Solution {
public:
    vector<int> sumZero(int n) {
        vector<int> v;
        if(n%2==0) {
            int count=-1*n;
            for(int i=count+1;i<n;i+=2) {
                v.push_back(i);
            }
        }
        else {
            int start=(n/2)*-1,end=n/2;
            for(int i=start;i<end+1;i++) {
                v.push_back(i);
            }
        }
        return v;
    }
};